utils/xh.py

```diff
@@ -28,3 +31,8 @@ def check_float_is_valid(val):
         return True
     except ValueError:
         return False
+
+
+def get_hospitals_podrazdeleniya(hospital_pk):
```

Comment:
Почему hospital_pk?
а не hospital?
и неплохо было бы сделать типизацию
```suggestion
def get_hospitals_podrazdeleniya(hospital: Hospital):
```

Reply:
И не понятно, почему оно в этом файле.
Лучше тогда рядом создать новый и назвать его podrazdeleniya.py